pkg/util/cache/mmap_hash.go

```diff
@@ -0,0 +1,54 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package cache
+
+import (
+	"errors"
+)
+
+// MaxValueSize is the largest possible value we can store.
+const MaxValueSize = 4080
```

Comment:
Where does it come from ?

Reply:
Copied from the `mmap_hash_linux`, which is based on hardware constraints & datastructure decisions that I don't expect to change across platforms.

https://github.com/DataDog/datadog-agent/blob/lally/exp-mem-metrefs/pkg/util/cache/mmap_hash_linux.go#L52